Library/Formula/ola.rb

```diff
@@ -2,8 +2,8 @@
 
 class Ola < Formula
   homepage "http://www.openlighting.org/ola/"
-  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.3/ola-0.9.3.tar.gz"
-  sha1 "f6a81087761218063a4bb8006b73ffa407cd0170"
+  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.5/ola-0.9.5.tar.gz"
+  sha1 "e9b0d8e4e3e4acd696cd7a0877d72d593727a7bc"
```

Comment:
Oh, just change the ` sha1 ` to a ` sha256 ` checksum. It's not a huge thing if you don't want to push another commit. In this case the ` sha256 ` is ` "7c24ac98e865b4c354a04563b88012e782205ffd932a05cf944273f6f5ea82ca" `.